Shared/SQL/init_db.sql

```diff
@@ -1,25 +1,25 @@
-drop database imperious;
-create database imperious;
-use imperious;
+drop database grupp01;
```

Comment:
You realise this isn't going to work when we do the weekly merge?
I guess we could merge this now, and change it right before we do the actual merge.